3-fleetscope/nonproduction.auto.example.tfvars

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+fleet_project_id   = "{FLEET_PROJECT}"
```

Comment:
We can do this on a 3-fleetscope refactor PR

I assume this would remove the need of symlinks and disabling the tflint, so the refactor of 3-fleetscope should also fix this

Reply:
We currently use symlinks is some of the other stages, so there must be some difference?